nf_core/lint/schema_lint.py

```diff
@@ -69,8 +71,10 @@ def schema_lint(self):
     # Lint the schema
     self.schema_obj = nf_core.schema.PipelineSchema()
     self.schema_obj.get_schema_path(self.wf_path)
+
     try:
         self.schema_obj.load_lint_schema()
+        self.schema_obj.validate_default_params()
```

Comment:
Nice! Though you're still calling it from the `nf-core lint` command which means that it won't run with the `nf-core schema lint` command.

I think that it maybe makes more sense to have it inside the `load_lint_schema()` function and just leave this linting file untouched?

Reply:
Ah, I have to admit I didn't even think about that subcommand :laughing: 

Sure, let's do that :+1:

Reply:
I really like it! But I suspect that I'm the only person who ever uses it.. Runs much faster than the main lint of course so is great if you're manually messing around with a schema. Which no-one ever does (hopefully).